tests/test_e2e.py

```diff
@@ -1,34 +1,145 @@
     assert res.returncode == 0
-    # ToDo evaluate result
     print(res.stdout)
 
-    # Clean up
-    res = subprocess.run(["illuminatio", "clean"], capture_output=True)
+    # delete test resources
+    res = subprocess.run(["kubectl", "delete", "-f", resource_manifest],
+                         capture_output=True,
+                         timeout=120)
+
     assert res.returncode == 0
     print(res.stdout)
 
-    # Clean up
-    corev1.delete_namespace(name=namespace)
+
+@pytest.mark.e2e
+def test_deny_all_traffic_to_an_application():
+    resource_manifest = "e2e-manifests/01-deny-all-traffic-to-an-application.yml"
+    results_yaml_file = "result.yml"
+    create_resources_and_run_illuminatio(resource_manifest, results_yaml_file)
+
+    # evaluate the results of illuminatio
+    with open(path.abspath(results_yaml_file)) as f:
+        yaml_document = yaml.safe_load_all(f)
+        for results_dict in yaml_document:
+            validate_illuminatio_was_successful(results_dict)
+            expected_dict = {'cases': {'01-deny-all:app=web': {'01-deny-all:app=web': {'-*': {'success': True}}}},
+                             'results': {'mappings': {'ports': {'01-deny-all:app=web': {
+                                                                '01-deny-all:app=web': {'-*': '-80'}}},
+                                         'toHost': {'01-deny-all:app=web': {'01-deny-all:app=web':
+                                                                            '01-deny-all:web'}}}}}
+            # exclude irrelevant information
+            del results_dict["runtimes"]
+            del results_dict["results"]["raw-results"]
+            # extract information from strings with random ids
+            from_host_dict = results_dict["results"]["mappings"]["fromHost"]
+            from_host_1 = from_host_dict["01-deny-all:app=web"]
+            del results_dict["results"]["mappings"]["fromHost"]
+            # compare the remaining dicts
+            assert results_dict == expected_dict
+            regex_pod_suffix = "[0-9a-z]{10}[-][0-9a-z]{5}"
+            regex_expected_start_string_1 = re.compile("01[-]deny[-]all:web[-]" + regex_pod_suffix)
+            # not None if regex matches
+            assert regex_expected_start_string_1.search(from_host_1)
+
+    cleanup_resources(resource_manifest)
+
+
+@pytest.mark.e2e
+def test_limit_traffic_to_an_application():
+    resource_manifest = "e2e-manifests/02-limit-traffic-to-an-application.yml"
+    results_yaml_file = "result.yml"
+    create_resources_and_run_illuminatio(resource_manifest, results_yaml_file)
+
+    # evaluate the results of illuminatio
+
+    with open(path.abspath(results_yaml_file)) as f:
+        yaml_document = yaml.safe_load_all(f)
+        for results_dict in yaml_document:
+            validate_illuminatio_was_successful(results_dict)
+            expected_dict = {'cases': {'default:app=bookstore': {
```

Comment:
Can we move this outside of the python code e.g. in a `results.yaml` that would make the code more readable.